backend/src/main/java/develup/domain/solution/Title.java

```diff
@@ -1,9 +1,46 @@
 package develup.domain.solution;
 
+import java.util.Objects;
+import develup.api.exception.DevelupException;
+import develup.api.exception.ExceptionType;
 import jakarta.persistence.Column;
 import jakarta.persistence.Embeddable;
-import jakarta.validation.constraints.NotBlank;
 
 @Embeddable
-public record Title(@NotBlank @Column(name = "title") String value) {
+public class Title {
+
+    @Column(name = "title")
+    private final String value;
+
+    protected Title() {
+        this.value = "";
+    }
```

Comment:
[Request Changes]

리플렉션 때문에 쓰는거라 내용물 필요 없을 것 같아요.

```suggestion
    protected Title() {
    }
```